<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite hostname="EPUALVIW1156" name="com.epam.tasks.firstTask.tests.GmailSendLetterTest" tests="9" failures="0" timestamp="4 Nov 2015 08:04:05 GMT" time="19.880" errors="5">
  <testcase name="setPassword" time="4.084" classname="com.epam.tasks.firstTask.tests.GmailSendLetterTest"/>
  <testcase name="setLogin" time="1.269" classname="com.epam.tasks.firstTask.tests.GmailSendLetterTest"/>
  <testcase name="sendMessage" time="0.000" classname="com.epam.tasks.firstTask.tests.GmailSendLetterTest">
    <error type="java.lang.Throwable" message="Method GmailSendLetterTest.sendMessage()[pri:0, instance:com.epam.tasks.firstTask.tests.GmailSendLetterTest@7dc36524] depends on not successfully finished methods">
      <![CDATA[java.lang.Throwable: Method GmailSendLetterTest.sendMessage()[pri:0, instance:com.epam.tasks.firstTask.tests.GmailSendLetterTest@7dc36524] depends on not successfully finished methods
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1063)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:108)
	at org.testng.TestRunner.privateRun(TestRunner.java:771)
	at org.testng.TestRunner.run(TestRunner.java:621)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:357)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:352)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:310)
	at org.testng.SuiteRunner.run(SuiteRunner.java:259)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1199)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1124)
	at org.testng.TestNG.run(TestNG.java:1032)
	at org.testng.remote.RemoteTestNG.run(RemoteTestNG.java:111)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:204)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:175)
]]>
    </error>
  </testcase> <!-- sendMessage -->
  <testcase name="createMessage" time="3.311" classname="com.epam.tasks.firstTask.tests.GmailSendLetterTest"/>
  <testcase name="openDraft" time="10.340" classname="com.epam.tasks.firstTask.tests.GmailSendLetterTest">
    <error type="org.openqa.selenium.TimeoutException" message="Timed out after 10 seconds waiting for url to contain &quot;drafts&quot;. Current url: &quot;https://mail.google.com/mail/#inbox&quot;
Build info: version: &#039;2.48.2&#039;, revision: &#039;41bccdd10cf2c0560f637404c2d96164b67d9d67&#039;, time: &#039;2015-10-09 13:08:06&#039;
System info: host: &#039;EPUALVIW1156&#039;, ip: &#039;10.25.14.105&#039;, os.name: &#039;Windows 8.1&#039;, os.arch: &#039;amd64&#039;, os.version: &#039;6.3&#039;, java.version: &#039;1.8.0_65&#039;
Driver info: org.openqa.selenium.firefox.FirefoxDriver
Capabilities [{applicationCacheEnabled=true, rotatable=false, handlesAlerts=true, databaseEnabled=true, version=41.0.2, platform=WINDOWS, nativeEvents=false, acceptSslCerts=true, webStorageEnabled=true, locationContextEnabled=true, browserName=firefox, takesScreenshot=true, javascriptEnabled=true, cssSelectorsEnabled=true}]
Session ID: 0961d6be-1ce1-4c1e-a78d-cec8f12cb635">
      <![CDATA[org.openqa.selenium.TimeoutException: Timed out after 10 seconds waiting for url to contain "drafts". Current url: "https://mail.google.com/mail/#inbox"
Build info: version: '2.48.2', revision: '41bccdd10cf2c0560f637404c2d96164b67d9d67', time: '2015-10-09 13:08:06'
System info: host: 'EPUALVIW1156', ip: '10.25.14.105', os.name: 'Windows 8.1', os.arch: 'amd64', os.version: '6.3', java.version: '1.8.0_65'
Driver info: org.openqa.selenium.firefox.FirefoxDriver
Capabilities [{applicationCacheEnabled=true, rotatable=false, handlesAlerts=true, databaseEnabled=true, version=41.0.2, platform=WINDOWS, nativeEvents=false, acceptSslCerts=true, webStorageEnabled=true, locationContextEnabled=true, browserName=firefox, takesScreenshot=true, javascriptEnabled=true, cssSelectorsEnabled=true}]
Session ID: 0961d6be-1ce1-4c1e-a78d-cec8f12cb635
	at org.openqa.selenium.support.ui.WebDriverWait.timeoutException(WebDriverWait.java:80)
	at org.openqa.selenium.support.ui.FluentWait.until(FluentWait.java:261)
	at com.epam.tasks.firstTask.tests.GmailSendLetterTest.openDraft(GmailSendLetterTest.java:93)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:85)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:659)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:845)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1153)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:108)
	at org.testng.TestRunner.privateRun(TestRunner.java:771)
	at org.testng.TestRunner.run(TestRunner.java:621)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:357)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:352)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:310)
	at org.testng.SuiteRunner.run(SuiteRunner.java:259)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1199)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1124)
	at org.testng.TestNG.run(TestNG.java:1032)
	at org.testng.remote.RemoteTestNG.run(RemoteTestNG.java:111)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:204)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:175)
]]>
    </error>
  </testcase> <!-- openDraft -->
  <testcase name="verifyDraftMessageWithOutOpen" time="0.000" classname="com.epam.tasks.firstTask.tests.GmailSendLetterTest">
    <error type="java.lang.Throwable" message="Method GmailSendLetterTest.verifyDraftMessageWithOutOpen()[pri:0, instance:com.epam.tasks.firstTask.tests.GmailSendLetterTest@7dc36524] depends on not successfully finished methods">
      <![CDATA[java.lang.Throwable: Method GmailSendLetterTest.verifyDraftMessageWithOutOpen()[pri:0, instance:com.epam.tasks.firstTask.tests.GmailSendLetterTest@7dc36524] depends on not successfully finished methods
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1063)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:108)
	at org.testng.TestRunner.privateRun(TestRunner.java:771)
	at org.testng.TestRunner.run(TestRunner.java:621)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:357)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:352)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:310)
	at org.testng.SuiteRunner.run(SuiteRunner.java:259)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1199)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1124)
	at org.testng.TestNG.run(TestNG.java:1032)
	at org.testng.remote.RemoteTestNG.run(RemoteTestNG.java:111)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:204)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:175)
]]>
    </error>
  </testcase> <!-- verifyDraftMessageWithOutOpen -->
  <testcase name="verifyDraftMessageWithOpenTo" time="0.000" classname="com.epam.tasks.firstTask.tests.GmailSendLetterTest">
    <error type="java.lang.Throwable" message="Method GmailSendLetterTest.verifyDraftMessageWithOpenTo()[pri:0, instance:com.epam.tasks.firstTask.tests.GmailSendLetterTest@7dc36524] depends on not successfully finished methods">
      <![CDATA[java.lang.Throwable: Method GmailSendLetterTest.verifyDraftMessageWithOpenTo()[pri:0, instance:com.epam.tasks.firstTask.tests.GmailSendLetterTest@7dc36524] depends on not successfully finished methods
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1063)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:108)
	at org.testng.TestRunner.privateRun(TestRunner.java:771)
	at org.testng.TestRunner.run(TestRunner.java:621)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:357)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:352)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:310)
	at org.testng.SuiteRunner.run(SuiteRunner.java:259)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1199)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1124)
	at org.testng.TestNG.run(TestNG.java:1032)
	at org.testng.remote.RemoteTestNG.run(RemoteTestNG.java:111)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:204)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:175)
]]>
    </error>
  </testcase> <!-- verifyDraftMessageWithOpenTo -->
  <testcase name="verifyDraftMessageWithOpenThema" time="0.000" classname="com.epam.tasks.firstTask.tests.GmailSendLetterTest">
    <error type="java.lang.Throwable" message="Method GmailSendLetterTest.verifyDraftMessageWithOpenThema()[pri:0, instance:com.epam.tasks.firstTask.tests.GmailSendLetterTest@7dc36524] depends on not successfully finished methods">
      <![CDATA[java.lang.Throwable: Method GmailSendLetterTest.verifyDraftMessageWithOpenThema()[pri:0, instance:com.epam.tasks.firstTask.tests.GmailSendLetterTest@7dc36524] depends on not successfully finished methods
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1063)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:108)
	at org.testng.TestRunner.privateRun(TestRunner.java:771)
	at org.testng.TestRunner.run(TestRunner.java:621)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:357)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:352)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:310)
	at org.testng.SuiteRunner.run(SuiteRunner.java:259)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1199)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1124)
	at org.testng.TestNG.run(TestNG.java:1032)
	at org.testng.remote.RemoteTestNG.run(RemoteTestNG.java:111)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:204)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:175)
]]>
    </error>
  </testcase> <!-- verifyDraftMessageWithOpenThema -->
  <testcase name="getGmailPage" time="0.876" classname="com.epam.tasks.firstTask.tests.GmailSendLetterTest"/>
</testsuite> <!-- com.epam.tasks.firstTask.tests.GmailSendLetterTest -->
